src/main/java/ru/itis/services/UserService.java

```
@@ -42,34 +48,29 @@ public void addUser(UserDto userDto) {
         usersRepository.save(newUser);
     }
 
-    public TokenDto login(UserDto userDto) {
-        Optional<User> userCandidate = usersRepository.findByLogin(userDto.getLogin());
-        String value = UUID.randomUUID().toString();
-        Token token = Token.builder()
-                .createdAt(LocalDateTime.now())
-                .expiredDateTime(LocalDateTime.now().plusSeconds(expiredSecondsForToken))
-                .value(value)
-                .build();
-        if (userCandidate.isPresent()) {
-            User user = userCandidate.get();
-            if (passwordEncoder.matches(userDto.getPassword(), user.getPassword())) {
-                token.setUser(user);
-                tokensRepository.save(token);
-                return TokenDto.from(token);
-            }
-        } else {
-            User defaultUser = User.getDefaultUser();
-            token.setUser(defaultUser);
-            return TokenDto.from(token);
-        }
-        return null;
+    public String createToken(User user) {
+        return Jwts.builder()
+                .claim("login", user.getLogin())
+                .claim("id", user.getId())
+                .signWith(SignatureAlgorithm.HS512, key)
+                .compact();
     }
 
     public Optional<User> findUserByToken(String token) {
         String userLogin = tokensRepository.findUsernameByValue(token);
         return usersRepository.findByLogin(userLogin);
     }
 
+    public TokenDto login(UserDto userDto) {
+        Optional<User> userCandidate = usersRepository.findByLogin(userDto.getLogin());
+
+        if (userCandidate.isPresent()) {
+            User user = userCandidate.get();
+            return new TokenDto(createToken(user));
+        }
+        return new TokenDto("Can not find such user");
```

You should throw an exception here.

---

fixed